tools/monorepo-utils/package.json

```diff
@@ -0,0 +1,35 @@
+{
+	"name": "monorepo-utils",
+	"version": "0.0.1",
+	"description": "WooCommerce Monorepo utility tooling.",
+	"author": "Automattic",
+	"homepage": "https://github.com/woocommerce/woocommerce",
+	"license": "GPLv2",
+	"repository": "woocommerce/woocommerce",
+	"dependencies": {
+		"chalk": "^4.1.2",
+		"commander": "^9.4.0",
+		"dotenv": "^10.0.0",
+		"ts-jest": "^29.1.0"
+	},
+	"devDependencies": {
+		"@commander-js/extra-typings": "^0.1.0",
+		"@types/jest": "^27.4.1",
+		"@types/node": "^16.18.18",
+		"@woocommerce/eslint-plugin": "workspace:*",
+		"eslint": "^8.32.0",
+		"jest": "^29.5.0",
+		"tslib": "^2.3.1",
```

Comment:
What are we using this for, out of curiosity?

Reply:
Nothing, will remove